extensions/editor.js

```diff
@@ -3,6 +3,7 @@
  */
 import './shared/public-path';
 import './shared/block-category';
+import './shared/notification-on-plan-upgrade';
```

Comment:
Maybe just `upgrade-notification` (or `plan-upgrade-notification`)? I didn't really want to set an example with `with-has-warning-is-interactive-class-names` :sweat_smile:

Reply:
Haha "plan-upgrade-notification" sounds great i think!